lib/socrata.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'soda/client'
+
+# This module holds the logic used in lib/tasks/socrata.rake
+module Socrata
+  class << self
+    # Creates a file in the current directory.
+    # Logs and Outputs the name of the file (timestamped).
+    # Returns nothing.
+    def export(file_format, collection)
+      log(:info, 'Exporting dataset...')
+
+      exporter = Decidim::Exporters.find_exporter(file_format || 'CSV')
+      export_data = exporter.new(collection, serializer).export
+      file_name = export_data.filename(collection.first.model_name.plural)
+      File.write(file_name, export_data.read)
+
+      log(:info, "File created: #{file_name}")
+    rescue StandardError => error
+      log(:error, error)
+    end
+
+    # Updates the remote Socrata dataset.
```

Comment:
"Increments the remote Socrata dataset."

Reply:
Thanks.